### PR TITLE
Add a link to the e-Mail form

### PR DESCRIPTION
**Pull request by Mailaender**
https://lablicate.com/about/contact to compensate for eclipse/chemclipse#1934. 
Icon is from https://www.eclipse.org/articles/Article-RCP-3/tutorial3.html

### CODE REVIEW

File: openchrom/plugins/net.openchrom.feature.branding/OSGI-INF/l10n/bundle.properties [hunk 1]
```diff
@@ -0,0 +1,13 @@
+###############################################################################
+# Copyright (c) 2024 Lablicate GmbH.
+#
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     Matthias Mail�nder - initial API and implementation
```
Comment (eselmeister) [inline openchrom/plugins/net.openchrom.feature.branding/OSGI-INF/l10n/bundle.properties:10]: Wrong encoding?

File: openchrom/plugins/net.openchrom.feature.branding/OSGI-INF/l10n/bundle_de.properties [hunk 2]
```diff
@@ -0,0 +1,13 @@
+###############################################################################
+# Copyright (c) 2024 Lablicate GmbH.
+#
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     Matthias Mail�nder - initial API and implementation
```
Comment (eselmeister) [inline openchrom/plugins/net.openchrom.feature.branding/OSGI-INF/l10n/bundle_de.properties:10]: Wrong encoding?

File: openchrom/plugins/net.openchrom.feature.branding/about.ini [hunk 3]
```diff
@@ -1,4 +1,4 @@
-featureImage=32x32-32.bmp
+featureImage=icons/32x32-32.bmp
 aboutText=OpenChrom� is a tool for the analysis and visualization of mass spectrometric and chromatographic data developed by Lablicate GmbH.\n\
```
Comment (eselmeister) [inline openchrom/plugins/net.openchrom.feature.branding/about.ini:2]: Wrong encoding?

Comment (eselmeister) [reply openchrom/plugins/net.openchrom.feature.branding/about.ini:2]: OK, here it looks ok. In the review it was odd.

**Comment by Mailaender** [comment]
I think the encoding is correct. It has to be `ISO-8859-1` because of @Eclipse and @github does not like it.

**Pull request merged**